static/bidder-info/yandex.yaml

```diff
@@ -0,0 +1,13 @@
+endpoint: "https://bs-metadsp.yandex.ru/prebid/{{.PublisherID}}?ssp-id=10500"
+endpointCompression: gzip
+maintainer:
+  email: "prebid@yandex-team.ru"
```

Comment:
@chernodub PBS has sent an email to verify above email address. Requesting to reply with "received" message on email thread

Reply:
Thanks! I asked the team to reply.

Reply:
received reply from Yandex